library/core/src/main/java/com/google/android/exoplayer2/util/FlacStreamMetadata.java

```diff
@@ -85,7 +89,8 @@ public FlacStreamMetadata(
       int channels,
       int bitsPerSample,
       long totalSamples,
-      List<String> vorbisComments) {
+      List<String> vorbisComments,
+      List<PictureFrame> pictureList) {
```

Comment:
`pictures` (the type already conveys that it's a list)

Reply:
Done